src/components/ui/experience.tsx

```
@@ -132,12 +140,21 @@ const Experience: React.FC<ExperienceProps> = ({ experiences = [] }) => {
     setWasValidated(false);
   };
 
-  const handleDelete = () => {
-    if (editingExperience) {
-      setExperienceList((prev) =>
-        prev.filter((exp) => exp.id !== editingExperience.id)
-      );
-      setIsModalOpen(false);
+  const handleDelete = async () => {
+    if (editingExperience && editingExperience.id) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)